doc/source/contributing.rst

```diff
@@ -525,7 +525,7 @@ supports both python2 and python3.
     so many stylistic issues are likely a result of automated transformation of the
     code.
 
-To install asv::
+To install asv (works also if you are using ``conda``)::
```

Comment:
but may be a nice clarification for people not familiar with 'conda' and
'pip'.

On Sun, Aug 30, 2015 at 1:13 PM, Jeff Reback <notifications@github.com>
wrote:

> In doc/source/contributing.rst
> <https://github.com/pydata/pandas/pull/10935#discussion_r38271077>:
>
> > @@ -525,7 +525,7 @@ supports both python2 and python3.
> >      so many stylistic issues are likely a result of automated transformation of the
> >      code.
> >
> > -To install asv::
> > +To install asv (works also if you are using ``conda``)::
>
> this is not necessary
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/10935/files#r38271077>.
>